kolibri/core/models.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+
+from kolibri.plugins import registry
+registry.initialize()
```

Comment:
I assume you put this here because you want it to run once, *after* settings have been loaded -- is that correct?